SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -222,6 +222,23 @@ JUPYTER_PATH=${ENV_PATH}/share/jupyter python -m ipykernel install --name "${ENV
 # We modify the already existing Python3 kernel with the kernel.json of the environment
 ln -f -s ${ENV_PATH}/share/jupyter/kernels/${ENV_NAME}/kernel.json /home/$USER/.local/share/jupyter/kernels/python3/kernel.json | tee -a ${LOG_FILE}
 
+# Pass Spark configurations to the environment kernel so it can be used in the notebooks
+if [ -n "${INSTALL_NXCALS}" ]; then
+    export SPARK_HOME="${ENV_PATH}/nxcals-bundle"
+    export PYSPARK_PYTHON="./environment/bin/python"
+    echo -e "export SPARK_HOME=\"${SPARK_HOME}\"\nexport PYSPARK_PYTHON=\"${PYSPARK_PYTHON}\"" >> /home/$USER/.bash_profile
```

Comment:
Please create two code blocks here, first one for the kernel and another one for the terminal, with a comment at the beginning of each block, e.g.

# Kernel configuration
# - SPARK_HOME: needed to point to the SPARK installation provided by the nxcals package
# - PYSPARK_PYTHON: ...
...

... 

# Terminal configuration
# Only SPARK_HOME and PYSPARK_PYTHON are needed, since `PATH` and `VIRTUAL_ENV` are already
# set when activating the environment in the terminal.  (I THINK THIS IS CORRECT, DOUBLE CHECK!)

...